### DISCUSSION TITLE
codice creato senza risultato

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#643] [Richieste di aiuto]
Buonasera,ho cercato di integrare al codice  "Argomenti.html.twig" la colonna dedicata ai compiti , ma quando mi reco nel registro vedo che il cambiamento non c'è ,continuo a vedere ciò che c'era prima.Allego ora il codice 

[codice colonna compiti .pdf](https://github.com/iisgiua/giuaschool/files/14130511/codice.colonna.compiti.pdf)

**Comment by sempliciniluigi89** [comment]
Mi correggo , sono riuscito a trovarlo nel registro , ma ci sono due problemi 

1.l'ho trovato facendo andando nella sezione argomenti quando volevo che apparisse  nella sezione "registro " insieme a "attività" "argomenti" "docenti""materia" etc

2.Appare nella sezione come la colonna dedicata ai compiti , ma non c'è nessun tasto che ti permette di entrare dentro e rendere la funzione effettiva ,mi sapreste cortesemente dire come si fa?

**Comment by trinko** [comment]
Scusa ma non puoi semplicemente modificare il template: la colonna "compiti" è un dato nuovo che non esiste nel database.
Devi modificare il database, aggiungere il codice SQL per la sua gestione, sia per l'inserimento che per l'estrazione, quindi potrai finalmente visualizzare qualcosa tramite i template.

Non è una modifica banale, richiede la modifica di più pagine PHP, del codice SQL e, come stavi facendo, del template.

**Reply by sempliciniluigi89** [reply]
Mi potrebbe spiegare per favore meglio questo passaggio : "Devi modificare il database, aggiungere il codice SQL per la sua gestione, sia per l'inserimento che per l'estrazione"

Grazie

**Comment by trinko** [comment]
Quando inserisci un nuovo dato, questo dovrà essere memorizzato in una tabella del database.
Siccome stai aggiungendo un dato che non è previsto nel database, dovresti anche modificare il database e di conseguenza modificare i comandi SQL per inserire il dato dalla pagina con il form.
Stessa cosa per leggere il nuovo dato: i comandi SQL attuali andrebbero modificati per consentire l'estrazione del dato (comando SELECT) dal database.
